pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -1,66 +1,217 @@
+		EngineConfig.Insecure = true
+		message := fmt.Sprintf("Would you like to import Engine CA cert locally from: %s ?", EngineConfig.PemURL)
+		err = survey.AskOne(
+			&survey.Confirm{
+				Message: message,
+				Default: false,
+				Help:    "In order to securly communicate with the oVirt engine, the certificate authority must be trusted by the local system.",
+			},
+			&importCACert,
+			nil)
+		if err != nil {
+			return EngineConfig, err
+		}
 	}
-	c.Insecure = !ovirtCertTrusted
 
-	if ovirtCertTrusted {
-		ovirtURL, err := url.Parse(c.URL)
+	// If users request, let's work to import Engine CA locally
+	if importCACert == true {
+		logrus.Info("Downloading Engine CA cert from Engine...")
+		tmpFile, err := ioutil.TempFile(os.TempDir(), "engine-")
 		if err != nil {
-			// should have passed validation, this is unexpected
-			return c, err
+			fmt.Println("Cannot create temporary file", err)
 		}
-		pemURL := fmt.Sprintf(
-			"%s://%s/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA",
-			ovirtURL.Scheme,
-			ovirtURL.Host)
-
-		err = survey.AskOne(&survey.Multiline{
-			Message: "oVirt certificate bundle",
-			Help:    fmt.Sprintf("The oVirt certificate bundle can be downloaded from %s.", pemURL),
-		},
-			&c.CABundle,
-			survey.ComposeValidators(survey.Required))
+		defer os.Remove(tmpFile.Name())
+
+		logrus.Debugf("CA cert temporary stored: %s", tmpFile.Name())
+
+		/* Build curl command */
+		OptionsCurl := []string{"-k", EngineConfig.PemURL, "-o", tmpFile.Name()}
+		_, err = execCommand(CurlCommand, OptionsCurl)
 		if err != nil {
-			return c, err
+			return EngineConfig, err
 		}
-	} else {
-		logrus.Warning("Communication with the oVirt engine will be insecure.")
+		logrus.Debugf("PEM file: %s", tmpFile.Name())
+
+		var content []byte
+		content, err = fileContent(tmpFile.Name())
+		if err != nil {
+			return EngineConfig, err
+		}
+		logrus.Debug(string(content))
+
+		/* Check if CA file already exists */
+		CaPath := fmt.Sprintf("/etc/pki/ca-trust/source/anchors/%s.pem", strings.ReplaceAll(EngineConfig.FQDN, ".", "-"))
+		if _, err := os.Stat(CaPath); err == nil {
+			return EngineConfig, err
+		}
+
+		/* Copy the CA to anchors */
+		CpCaToAnchros := []string{CpCommand, tmpFile.Name(), CaPath}
+		_, err = execCommand(SudoCommand, CpCaToAnchros)
+		if err != nil {
+			return EngineConfig, err
+		}
+
+		/* chmod to allow non root users to read the cert */
+		PermsPemfile := []string{ChmodCommand, "0644", CaPath}
+		_, err = execCommand(SudoCommand, PermsPemfile)
+		if err != nil {
+			return EngineConfig, err
+		}
+
+		/* Update CA database */
+		UpdateCaTrust := []string{"/usr/bin/update-ca-trust"}
+		_, err = execCommand(SudoCommand, UpdateCaTrust)
+		if err != nil {
+			return EngineConfig, err
+		}
+		EngineConfig.Insecure = false
+		logrus.Infof("%s imported with success!", CaPath)
+	}
+
+	if EngineConfig.Insecure == true {
+		logrus.Warning("Communication with the Engine will be insecure.")
```

Comment:
I think that you should explain why, trust list and so on...

Reply:
This will be a note after this question: https://github.com/openshift/installer/pull/3663/files/1a124d4123713be3167827c3ef677b7d94466f23#diff-322dd8bb06502c49e946bd5fd3b0aad8R140

Reply:
ok so maybe as info?
I'm just trying to think of looking into the logs and seeing such an error it can be alarming...
So I think that we should explain more or mark it as info since it is something that the user chooses.

Reply:
IMO, as user **decided to go insecure**, we must raise as warning instead of info.